tests/desktop/consumer_pages/test_home_page.py

```diff
@@ -52,26 +52,31 @@ def test_that_verifies_categories_menu(self, mozwebqa):
 
     @pytest.mark.sanity
     @pytest.mark.nondestructive
-    def test_opening_every_category_page_from_categories_menu(self, mozwebqa):
+    def test_opening_category_pages_from_categories_menu(self, mozwebqa):
+        """Open the first 3 category pages and check the first 3 apps on those pages."""
 
         home_page = Home(mozwebqa)
         home_page.go_to_homepage()
 
-        for i in range(home_page.category_count):
+        categories = home_page.categories.items
+        for c in range(3):
```

Comment:
I added a docstring at the top of the test that explains that we are checking the first 3 categories and the first 3 apps on the page, so I thought that might be sufficient. I can also add one here if you think that's needed.